lib/plugins/sfDoctrinePlugin/lib/task/sfDoctrineBuildTask.class.php

```diff
@@ -185,61 +187,63 @@ protected function execute($arguments = [], $options = [])
             }
         }
 
-        if (self::BUILD_SQL == (self::BUILD_SQL & $mode)) {
```

Comment:
After `BUILD_DB` are the three tasks which are relatively safe for production:
- `BUILD_MODEL`
- `BUILD_FORMS`
- `BUILD_FILTERS`

Those are fine, so have been left untouched.

From here down, however, are more dangerous ones which could allow for corruption of the database if mishandled. It looks like a lot has changed, but I've just wrapped all three conditional tests with an `if (!$is_production) {...}` so they're all impossible to run in production too.